scripts/west_commands/build.py

```diff
@@ -445,6 +461,14 @@ def _run_cmake(self, board, origin, cmake_opts):
         if user_args:
             cmake_opts.extend(shlex.split(user_args))
 
+        config_sysbuild = config_getboolean('sysbuild', False)
```

Comment:
Which Kconfig defines the `CONFIG_SYSBUILD`?

Reply:
this is not a Kconfig setting, but a `west config` setting.

This allows a user to opt-in to use sysbuild per default, that means instead of:
```
west build --sysbuild ...
```
each time, then a user can do:
```
west config build.sysbuild True
```
and then simply do:
```
west build ...
```

See also the docs: https://github.com/zephyrproject-rtos/zephyr/blob/ba85e4cb39b1c5ba35bb0efdca5b5061e1d2a2fb/doc/guides/sysbuild/index.rst#building-an-application-with-sysbuild